docs/source/autodoc.rst

```diff
@@ -0,0 +1,16 @@
+.. _autodoc:
+
+All content
+======================================
+
+This is the list of all content for the *torchquad* project.
+We are currently implementing new methods in our library.
+
+For the code, please see the `code module <https://torchquad.readthedocs.io/en/latest/_modules/index.html>`_ 
```

Comment:
This is a little ambiguous, the former is the code documentation not the code, right? :)

Reply:
The former is "All modules for which code is available", so actual code

Reply:
I got Pablo's point when he says that it is a little bit ambiguous. One can think you are pointing to the codebase, instead of the documentation. Maybe you can solve the ambiguity as follows: "Here you can find the All modules for which code is available." or similar.

Reply:
I am in fact pointing to the code base